Engine/source/windowManager/platformWindowMgr.h

```diff
@@ -136,6 +136,10 @@ class PlatformWindowManager
    /// This method indicates to created windows to show as normal.
    virtual void setDisplayWindow(bool set){}
 
+   /// Checks if DWM is running (Windows only)
+   virtual bool isDesktopCompositionEnabled() { return false; }
+   virtual void updateDesktopCompositionState() {}
```

Comment:
Is there any way we could avoid adding Windows-specific capability to a general class?